kibble/render/render.go

```diff
@@ -152,6 +175,63 @@ func createLanguage(cfg *models.Config, lang string, locale string) *models.Lang
 		Code:               lang,
 		Locale:             locale,
 		IsDefault:          (lang == cfg.DefaultLanguage),
-		DefinitionFilePath: fmt.Sprintf("%s.all.json", locale),
+		DefinitionFilePath: fmt.Sprintf("%s.all.json", lang),
+	}
+}
+
+func overrideLanguages(site *models.Site, cfg *models.Config) error {
+	if site.Toggles["translations_api"] {
+
+		languages, err := api.LoadAllLanguages(cfg)
+		if err != nil {
+			log.Errorf("Failed to get languages: %s", err)
+			return err
+		}
+
+		if cfg.Languages == nil {
+			cfg.Languages = make(map[string]models.LanguageConfig)
+		}
+		cfg.DefaultLanguage = strings.Replace(languages.DefaultLanguage["code"], "_", "-", -1)
```

Comment:
Are we on an old version of Go that doesn't have [`ReplaceAll()`](https://pkg.go.dev/strings#ReplaceAll)?

Reply:
I note #91 is pending.

Reply:
Changed to be done we retrieve API data uses ReplaceAll in api/translations.go